### PR TITLE
11Jul

### PR DESCRIPTION
**Pull request by kums78**
Appy Please check now ya

### CODE REVIEW

File: bundle_the_input_dir.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import tarfile
+
+file_or_dir = sys.argv[1]
+
+if os.path.exists(file_or_dir) & os.path.isdir(file_or_dir):
+	bundle_name = file_or_dir + ".tar.gz"
+	with tarfile.open(bundle_name, "w:gz") as tar:
+		tar.add(file_or_dir)
+	print("Argument {} is directory and bundle {} is created".format(file_or_dir, bundle_name))
+
```
Comment (lohit) [inline bundle_the_input_dir.py:14]: No blank line between if/elif/else

File: bundle_the_input_dir.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env python3
+
+import sys
+import os
+import tarfile
+
+file_or_dir = sys.argv[1]
```
Comment (lohit) [inline bundle_the_input_dir.py:7]: You are not checking if the correct number of arguments are being passed by the user

File: bundle_the_input_dir.py (hunk 1 above)
Comment (kums78) [reply bundle_the_input_dir.py:14]: OK. Other than that?

File: bundle_the_input_dir.py (hunk 2 above)
Comment (kums78) [reply bundle_the_input_dir.py:7]: When nothing is passed, it says file not found na

Comment (lohit) [reply bundle_the_input_dir.py:7]: `def usage():
    print("{} {}".format(sys.argv[0], "filesystem_path"))
    sys.exit(200)

def main():
    if len(sys.argv) < 2:
        usage()
    else:
        process(sys.argv[1])`

Comment (kums78) [reply bundle_the_input_dir.py:7]: OK got it ya.

**Comment by kums78** [comment]
Appy what is next problem ya?

**Pull request closed without merging**